src/lib/api-types.ts

```diff
@@ -63,6 +63,11 @@ export interface Student {
 	Views: View[];
 }
 
+export interface AuthedStudent {
```

Comment:
Perhaps instead of `Authed` which is not a word, it could be `AuthenticatedStudent`